CBinding/Gui/GeneralOptionsPanel.cs

```diff
@@ -42,7 +42,7 @@ namespace CBinding
 {
 	public partial class GeneralOptionsPanel : Gtk.Bin
 	{
-		ICompiler default_c_compiler;
+		public ICompiler default_c_compiler;
```

Comment:
Don't make fields public, use properties. And properties should use PascalCase.

That said, this shouldn't be exposed anyways... see my comment above about PropertyService.